docs/API/InstrumentFormat.md

```diff
@@ -18,6 +23,37 @@ this standard without breaking backwards compatibility. Instruments MAY implemen
 the instrument format described below and the accompagnying rules format in the same
 JSON object, or they MAY be used independently.
 
+Language specifications can be added for fields that display text (LongName, Description, Values, Labels) in the following format:
+
+```js
+...
+    "RegularText" : {
+        "lang-1" : "text in lang 1",
+        "lang-2" : "text in lang 2",
+        ...
+    },
+    "Values" : {
+        "lang-1" : {
+            "val1" : "0"
+            "val2" : "1"
+        },
+        "lang-2" : {
+            "val1" : "10"
+            "val2" : "9"
+        },
+        ...
+    },
+...
+```
+
+The text displayed will depend on the candidate's preferred language parameter. The default is `"en-ca"`.
+
+Additionaly, we support HTML tags in labels and descriptions that will be rendered with the element.
```

Comment:
This is unsafe and not portable. If there ever were a need for HTML-like markup, it would at the very minimum need to be an explicit whitelist, but even then I don't think it's a good idea to tie the semantics of the instrument description to an HTML renderer.

Reply:
I agree.
CAP requires it because some of the instruments we import have HTML tags embedded in the string they provide as a description or label which need to be rendered, but for LORIS it can be removed.